packages/tinacms-provider-github/README.md

```diff
@@ -0,0 +1,38 @@
+he Github Git Provider handles saving and deleting content to github. It can be used as a prop to the `createDatabase` function.
```

Comment:
@logan-anderson should this package be named *-gitprovider-* or *-git-provider-*?

Reply:
I am ok with either one

Reply:
ok - so am I, you pick :) I just felt provider was too generic...

Reply:
Pushing up an update that renames it to -gitprovider-